Demo.Service/_Base/BaseCrudAsyncController.cs

```diff
@@ -185,6 +199,7 @@ public virtual async Task<ActionResult> DeleteAsync(TPrimaryKey id)
 
             if (entity == null)
             {
+                Log.Warning($"[Delete Action]: Can't find Id { id }");
                 return NotFound($"Can't find Id { id }");
```

Comment:
`Not Found` is a term wrapped status code 404. This means the URL what we passed is wrong. Use `No Content` or something like that.